.agent/skills/vscode-server/scripts/make-demo-fixture.sh

```
@@ -0,0 +1,220 @@
+  case "$1" in
+    --server) mode="server"; shift ;;
+    -h|--help) sed -n '3,20p' "$0" | sed 's/^# \{0,1\}//'; exit 0 ;;
+    -*) echo "ERROR:unknown option '$1'"; exit 1 ;;
+    *) target_dir="$1"; shift ;;
+  esac
+done
+target_dir="${target_dir:-tmp/bd-demo-fixture}"
+prefix="orbit"
+
+case "$target_dir" in
+  /*) target_dir="$target_dir" ;;
```

Shellcheck (0.11.0)

[info] This variable is assigned to itself, so the assignment does nothing.

(SC2269)